data/org.ayatana.indicator.power.gschema.xml.in

```diff
@@ -1,24 +0,0 @@
-<schemalist>
```

Comment:
You have deleted this file, instead of doing a rename + replace.

Reply:
Ah, forgot to add this to Git. Amended now.